conda_forge_tick/auto_tick.xsh

```diff
@@ -410,7 +410,10 @@ def main(args=None):
                 break
             $PROJECT = attrs['feedstock_name']
             $NODE = node
-            logger.info('%s IS MIGRATING %s', migrator.__class__.__name__.upper(),
+            m_name = migrator.__class__.__name__.upper()
+            if m_name == 'REBUILD':
+                m_name = migrator.name.upper()
```

Comment:
Maybe all migrators classes should have a name attr/property, which just defaults to `__class__.__name_`_, so that you don't have to put this logic everywhere

Reply:
And can just use migrator.name.

Reply:
This would break our current name for the Arch rebuild

Reply:
But is a better design overall.

Reply:
Can we put that into its own PR? It seems like that might be a breaking change, or at the very least require a description kwarg added to the migrator.

Reply:
It seems like there should be a name property, which takes care of this

Reply:
The ``name`` is used alongside the ``migrator_name`` (aka class name) in the PRed_json keys.

Reply:
Maybe there should be a different attribute / property for this then

Reply:
Maybe, or a migration of the graph itself.